compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -202,9 +202,15 @@ public static String rewriteVirtualCallTypeName(String value, BType objectType)
         objectType = getImpliedType(objectType);
         // The call name will be in the format of`objectTypeName.funcName` for attached functions of imported modules.
         // Therefore, We need to remove the type name.
+        Name originalName = objectType.tsymbol.originalName;
         if (!objectType.tsymbol.name.value.isEmpty() && value.startsWith(objectType.tsymbol.name.value)) {
```

Comment:
Would it improve readability if we extract out `objectType.tsymbol.name.value` and  `objectType.tsymbol.originalName` into variables as follows?
```
String typeName = objectType.tsymbol.name.value;
String originalName = objectType.tsymbol.originalName != null ? objectType.tsymbol.originalName.value : "";
```